src/dmqproto/client/mixins/NeoSupport.d

```diff
@@ -625,7 +626,9 @@ template NeoSupport ( )
     {
         this.neo = new Neo(config, Neo.Settings(conn_notifier, new SharedResources));
         this.blocking = new TaskBlocking;
-        this.neo.enableSocketNoDelay();
+
+        static if (!hasFeaturesFrom!("swarm", 4, 7))
```

Comment:
Hm I just noticed the swarm version you're checking here doesn't match the one mentioned in the commit message...?

Reply:
Oh, you're right. It should be v5.1.0. However, this also works because v5.1.0 is the first one to have features from v4.7.0. The commit message must say `v5.1.0`, because you can't use `swarm v4.x.x` with this version. I suggest we leave it in here, though, to be synchronized across other submodules. Alternatively, I can just add PRs to other submodules (and release dhtproto again), but I don't think it's worth it.

Reply:
:+1:

Reply:
Updated.